{"name": "Проверка https://dev-api.getprime.pro/api/v1/auth - invalid-api-key", "status": "passed", "description": "Негативная проверка с передачей неверного ключа ", "steps": [{"name": "Проверка https://dev-api.getprime.pro/api/v1/auth на код 401", "status": "passed", "start": 1753954318027, "stop": 1753954318216}, {"name": "Проверка текста message. Ожидали текст 'Вы не авторизованы.                 Пожалуйста, выполните вход и повторите попытку.' Получили 'Вы не авторизованы. Пожалуйста, выполните вход и повторите попытку.'", "status": "passed", "start": 1753954318408, "stop": 1753954318408}], "start": 1753954318027, "stop": 1753954318409, "uuid": "b864e863-7a0e-46c7-a60e-4370a3d0922f", "historyId": "f6aad7747424500f0f55fb65296e0431", "testCaseId": "f6aad7747424500f0f55fb65296e0431", "fullName": "tests.test_api.TestApiGet#test_auth_wrong_key", "labels": [{"name": "tag", "value": "apitest"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApiGet"}, {"name": "host", "value": "Robotek3000"}, {"name": "thread", "value": "2780-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}